docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -3,87 +3,147 @@
 
 preview::[]
 
-The Observability AI Assistant is a large language model (LLM) integration that helps Elastic Observability users by adding context, explaining errors and messages, and suggesting remediation in the {observability} interface. You can connect the Observability AI Assistant to either the OpenAI or Microsoft Azure LLM service. 
+The AI Assistant uses generative AI, powered by a {kibana-ref}/gen-ai-action-type.html[connector] for OpenAI or Azure OpenAI Service, to provide {observability} users with:
 
-You can find Observability AI Assistant prompts throughout {observability}:
+* *Contextual insights* — open prompts throughout {observability} that explain errors and messages and suggest remediation. 
+* *Chat* —  have conversations with the AI Assistant. Chat uses function calling to request, analyze, and visualize your data.
 
 [role="screenshot"]
-image::images/obs-assistant.gif[Observability AI assistant preview, 90%]
-
-[IMPORTANT]
-====
-Observability AI Assistant is in technical preview, and its capabilities are still developing. Users should leverage it sensibly as the reliability of its responses might vary. Always cross-verify any returned advice for accurate threat detection and response, insights, and query generation.
```

Comment:
I might be missing something, but we seem to have replaced the warning about technical preview with a generic one that says ```This functionality is in technical preview and may be changed or removed in a future release. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.```

^ Alongside this warning we want to include the warning that was removed from here on the unreliable nature of the advice and the need to cross verify it.